using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+
+</div>
+
+If you can't or don't want to use your own existing REDCap project, you can clone a sample project from us.  We've taken data from the University of California, Irvine Machine Learning Repository, specifically, their dataset on [cervical cancer risk factors](https://archive.ics.uci.edu/ml/datasets/Cervical+cancer+%28Risk+Factors%29), and made that data into a REDCap project.  Here are the steps to take our REDCap project and make it yours!
+
+1. Download our [REDCap xml file](media/cervical_cancer_factors.REDCap.xml), saving it to a place you'll be able to find it (like your Downloads folder).  We suggest doing a right click on the link and choosing "Save Link As", or "Download Linked File As".
+2. Log in to your REDCap server and choose "New Project".
+3. Fill out the Project title, Project purpose, Assign project to Folder, and Project notes fields as you like.  We called our project "API Testing" but that's only a suggestion.
+4. Choose "Upload a REDCap project XML file" as the Project Creation option.  
+5. Click "Choose File", and in the file picker, navigate to where you saved our XML file.
+
+An image below shows what you might see after steps 1-5, although your institution may have different colors, layouts, etc. on their page.
+
+![New project form in REDCap](media/new_project_from_xml.png)<!-- style = "border :1px solid rgb(var(--color-highlight)); max-width:800px;" -->
+
+6. At the bottom of the page, click "Create Project".  It may take a few minutes to digest that giant XML file!
+
+Congratulations, now you have a project that you created, which means you have User Rights capabilities.  That will come in handy in our next step!
+
+### Enabling the API
+
+In REDCap, the project owner (or people with user rights) have to explicitly give permission to use the REDCap API to project users who need that capability. In fact, when you create a new REDCap project, REDCap does not provide you with these permissions by default, even though you're the project owner! Take a look at your REDCap project and look on the left hand side of the screen to look at the menu of options. Chances are, you won't see anything that says "API". If that's the case, don't worry. We're going to walk you through how to give yourself API access.
+
+Open your REDCap database (one that belongs to you or in which you have the ability to change user rights). In the left side of the screen, choose "User Rights" from the list of applications.
+
+![List of applications which includes the "User Rights" application](media/user_rights.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+The User Rights panel will open in the main part of your screen. Click on your user id (or your role) and choose "Edit User Privileges". Give yourself API Export access, as shown in the image below.  For now, we'll leave API Import/Update turned off, so that you feel secure knowing you can't accidentally upload anything that could mess up your data.
```

Comment:
```suggestion
The User Rights panel will open in the main part of your screen. Click on your user id (or your role) and choose "Edit User Privileges." Give yourself API Export access, as shown in the image below.  For now, we'll leave API Import/Update turned off, so that you feel secure knowing you can't accidentally upload anything that could mess up your data.
```